lib/bundle-validator.js

```diff
@@ -86,31 +103,45 @@ function validateThemeConfiguration(callback) {
 
     // Validate against the theme registry config schema
     if (this.isPrivate) {
-        validationSchema = './privateThemeConfig.schema.json';
+        validationSchema = './schemas/privateThemeConfig.json';
     }
 
-    validation = v.validate(this.themeConfig.getRawConfig(), require(validationSchema));
-
-    if (validation.errors && validation.errors.length > 0) {
-        errorMessage = 'Your theme\'s config.json has errors:'.red;
-        validation.errors.forEach(error => {
-            errorMessage += '\r\nconfig'.red + error.stack.substring(8).red;
-        });
-
-        return callback(new Error(errorMessage));
+    const error = validateJsonSchema('config', require(validationSchema), this.themeConfig.getRawConfig());
+    if (error) {
+        return callback(error);
     }
 
     callback(null, true);
 }
 
+/**
+ *
+ * @param type
+ * @param schema
+ * @param data
+ * @returns {Error}
+ */
+function validateJsonSchema(type, schema, data) {
+    let errorMessage;
+    const validator = new Validator(JsonSchemaValidatorOptions);
+    validator.validate(schema, data);
+    if (validator.errors && validator.errors.length > 0) {
+        errorMessage = 'Your theme\'s ' + type + '.json has errors:';
+        validator.errors.forEach(error => {
+            errorMessage += '\r\n' + type + error.dataPath + " " + error.message;
```

Comment:
Would be better to use `os.EOL` and anywhere we have `\n` or `\r\n` since it is OS-specific